TWLight/users/templates/users/preferences.html

```diff
@@ -34,17 +34,43 @@ <h3>Preferences</h3>
       <div class="col-xs-12 col-sm-9">
         <form method="post" action="{% url 'password_change' %}">
           {% csrf_token %}
-          {{ password_form | crispy }}    
+          {{ password_form | crispy }}
           <button class="btn btn-default" type="submit">
             {% trans "Change password" %}
           </button>
-        </form>    
+        </form>
       </div>
     </div>
 
     <hr />
   {% endif %}
 
+  {% if user.userprofile.terms_of_use %}
+    <div class="row clearfix">
+      <div class="col-xs-12 col-sm-3">
+        {% comment %} Translators: This is the heading of the section of a user's
+        profile where they disagree their terms of use. {% endcomment %}
+        <strong>{% trans "Terms of use" %}</strong>
+      </div>
+      <div class="col-xs-12 col-sm-9">
+        <form method="post" action="{% url 'terms' %}">
+          {% csrf_token %}
+          {{ terms_form | crispy }}
+
+          <button class="btn btn-default" type="submit">
+             {% comment %} Translators: This is a button for updating a user's term of use. {% endcomment %}
```

Comment:
_terms_ :)

Reply:
Could be slightly tweaked to something like, _This text labels a button which, when clicked, updates a user's acceptance to the site's terms of use._

Reply:
Updated.  Please let me know if you prefer a different tone or verbiage.